tox.ini

```diff
@@ -631,6 +631,45 @@ commands =
     bash -c 'cd packages/grid/helm/repo && \
         helm repo index . --url https://openmined.github.io/PySyft/helm'
 
+
+[testenv:dev.k8s.ready]
+description = Check readiness of k8s deployement
+changedir = {toxinidir}/packages/grid
+allowlist_externals =
+    bash
+    tox
+    curl
+setenv =
+    CLUSTER_NAME = {env:CLUSTER_NAME:syft}
+    CLUSTER_HTTP_PORT = {env:NODE_PORT:8080}
+; Usage for posargs: names of the relevant services among {frontend backend proxy mongo seaweedfs registry}
+commands =
+    bash -c "env; date; k3d version"
+
+    # Frontend
+    bash -c "if echo '{posargs}' | grep -q 'frontend'; then echo 'Checking readiness of frontend'; ./scripts/wait_for.sh service frontend --context k3d-$CLUSTER_NAME --namespace syft; fi"
+    bash -c "if echo '{posargs}' | grep -q 'frontend'; then echo 'Checking readiness of frontend'; (kubectl logs service/frontend --context k3d-$CLUSTER_NAME --namespace syft -f &) | grep -q -E 'Network:\s+https?://[a-zA-Z0-9.-]+:[0-9]+/' || true; fi"
```

Comment:
I think this will echo the 'Checking for readiness of frontend' twice. We should omit echo in the second command, or merge these two command into a single command. Something like:
```bash
bash -c "if echo '{posargs}' | grep -q 'frontend'; then echo 'Checking readiness of frontend'; ./scripts/wait_for.sh service frontend --context k3d-$CLUSTER_NAME --namespace syft && (kubectl logs service/frontend --context k3d-$CLUSTER_NAME --namespace syft -f &) | grep -q -E 'Network:\s+https?://[a-zA-Z0-9.-]+:[0-9]+/' || true; fi"
```